src/cells/model.ts

```diff
@@ -357,8 +357,8 @@ class BaseCellModel implements IBaseCellModel {
   type: CellType;
 
   private _input: IInputAreaModel = null;
-  private _tags = '';
-  private _name = '';
+  private _tags = '{}';
+  private _name = '{}';
```

Comment:
Should the default `_name` be `'{}'` or `''`? The empty string makes more sense to me.